### ISSUE TITLE
Support RAK7246 gateway

### ISSUE THREAD

**Issue by ama9910** [balenalabs/basicstation#31]
Would it be possible to provide support for the RAK7246 gateway, which is a Raspberry Pi Zero W with a RAK2246 (SX1308) concentrator.

Currently building fails with
`setup.gmk:65: No toolchain for platform 'rpi' and local arch is not ''`

Presumably due to the arm32 architecture.

**Comment by mpous** [comment]
@ama9910 yes you can contribute on introducing the SX1308 into the project. Basically you would need to edit the setup.gmk and the start.sh files plus some others, but should be easy! Let me know if i can help you more!
